### PR TITLE
improve toolbar accessibility for small screens

### PR DESCRIPTION
**Pull request by theInternauts**
- improve toolbar accessibility for small screens
This addresses:
- issue #35 and #26 

Screen Shot 2022-04-14 at 00 04 35

### CODE REVIEW

**Review by zspencer — APPROVED** [review]
Looks great! Any chance you can add this to the CHANGELOG.md so it's clear what is changing in the upcoming release?

**Comment by theInternauts** [comment]
I updated the changelog.  I'm not in love with the format.  I think it's too verbose. But, that's probably a minority opinion.

**Comment by zspencer** [comment]
Hmm, I'm really confused about the updates you made to the changelog. It's likely because the Major and minor version numbers are drifting.

Here's all I really care about from the Changelog:

1. Any should include all user-or-contributor-facing changes of significance, and ideally include links to the PR that adds the change.
2. It must be ordered so the most-recent changes are at the top.
3. It must be grouped by Version Number.

It doesn't need to be verbose, it can literally be:


```
## Current Release
* Make to make it easier to swipe up on smaller screens.
* Switch to xCode 13 for our development environment

## 3.3.??? (I don't know what the most recent release is)
* Update Copy to match Polaroid's trademark requirements

## ???
* ???
```

Basically, a place for contributors to see what has been added to the current release, even if we don't decide to give that Release a version number until we actually decide it's time to do a release (i.e. we could bump to 3.4 for this, or go 3.3.5 if we decide to just release the bug-fixes and not add any other enhancements)

Moment Park is your project, you get to decide what you want to do with it. I'm sharing what I've seen work well where people work independently and highly asynchronously. Yes, it means a higher degree of rigor and discipline around communication, but that's how we get good at working without having to be in the same place at the same time.
